Automattic/Gravatar-SDK-iOS
# Alt text editor

Closes #535 

### Description

Adding the option to edit the Alt Text of an avatar.

![CleanShot 2024-12-10 at 15 23 47](https://github.com/user-attachments/assets/39ee42c7-f789-4c1e-b846-c9100165c66d)


I've spent a lot of time trying to make the content-height bottom sheet work unsuccessfully.
So I decided to make this first PR with it showing full screen. As the keyboard appears automatically, it's not too bad.

I've also made small modifications to the `.gravatarNavigation` modifier, to share common behaviors.


### Testing Steps

- Open Quick Editor on the demo app.
- Tap the Alt Text option on an avatar's contextual menu.
- Edit the alt text and save.
  - New alt text should be reflected on web.
